_books/ion-1-1/src/modules/system_module.md

```diff
@@ -7,23 +7,20 @@ leverages the system symbol table, the text encoding that users typically intera
 The system macros are more visible, especially to authors of macros.
 
 This chapter catalogs the system-provided symbols and macros.
-The examples below use unqualified names, which works assuming no other macros with the same name are in scope. The unambiguous form `$ion::macro-name` is always available to use in the [template definition language](../macros/defining_macros.md#template-definition-language-tdl).
+The examples below use unqualified names, which works assuming no other macros with the same name are in scope.
+The unambiguous form `$ion::macro-name` is always available to use.
 
 ### Relation to local symbol and macro tables
 
 In Ion 1.0, the system symbol table is always the first import of the local symbol table.
-However, in Ion 1.1, the system symbol and macro tables have a system address space that is distinct from the local address space.
-When starting an Ion 1.1 segment (i.e. immediately after encountering an `$ion_1_1` version marker),
-the local symbol table is prepopulated with the system symbols[^note0]<a name="footnote-0"></a>. 
-The local macro table is also prepopulated with the system macros.
-However, the system symbols and macros are not permanent fixtures of the local symbol and macro tables respectively.
-
+However, in Ion 1.1, the system symbol and macro tables have a system address space that is distinct from the 
+local address space, but can optionally be included in the user address space.
```

Comment:
We have a lot of names for this--[local | user | application] address space. We should standardize on one and put it in the glossary.